docs/content/docs/connectors/datastream/formats/mongodb.md

```diff
@@ -0,0 +1,33 @@
+---
+title:  "MongoDb"
+weight: 4
+type: docs
+aliases:
+- /dev/connectors/formats/mongodb.html
+- /apis/streaming/connectors/formats/mongodb.html
+
+---
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+# MongoDB format
```

Comment:
Is MongoDB a format or a connector? I would expect the latter?

Reply:
it is a connector indeed I was too quick sorry, thanks !